vscode/tsconfig.json

```diff
@@ -1,14 +1,17 @@
 {
   "compilerOptions": {
-    "module": "Node16",
-    "target": "ES2023",
-    "lib": ["ES2023", "WebWorker", "DOM"],
-    "jsx": "react",
+    "module": "commonjs",
```

Comment:
Is that actually right?  Does vite output esm or commonjs for the project?

Reply:
Actually I don't really even see where the not webview-ui stuff is being compiled/packaged...

Reply:
Thats being done via the task.json using the tsc build. We still don't have a bundler with this approach but [esbuild](https://github.com/continuedev/continue/blob/main/extensions/vscode/scripts/esbuild.js) could work for us.

Reply:
Ended up moving back to webpack for now & following your suggestion to moving webview to its own dir. Next step would definitely be to use a workspace or something similar to tackle2-ui.